docs/app/pages/components/sections/panels/item-display-panel-inline/item-display-panel-inline.component.ts

```diff
@@ -0,0 +1,384 @@
+import { ColorService } from './../../../../../../../src/services/color/color.service';
+import { Component } from '@angular/core';
+import { DocumentationSectionComponent } from '../../../../../decorators/documentation-section-component';
+import { BaseDocumentationSection } from '../../../../../components/base-documentation-section/base-documentation-section';
+import 'chance';
+import { IPlunk } from '../../../../../interfaces/IPlunk';
+import { IPlunkProvider } from '../../../../../interfaces/IPlunkProvider';
+import { ItemDisplayPanelComponent } from '../../../../../../../src/index';
+
+@Component({
+    selector: 'uxd-item-display-panel-inline-component',
+    templateUrl: './item-display-panel-inline.component.html',
+    styleUrls: ['./item-display-panel-inline.component.less'],
+    host: {
+        '(window:keydown.arrowup)': 'upArrow($event)',
+        '(window:keydown.arrowdown)': 'downArrow($event)'
+    }
+})
+@DocumentationSectionComponent('ComponentsItemDisplayPanelInlineComponent')
+export class ComponentsItemDisplayPanelInlineComponent extends BaseDocumentationSection implements IPlunkProvider {
+
+    visible: boolean = false;
```

Comment:
Some of these properties are never changed in the class, in which case just set them in the HTML rather than having variables for them